docs/source/contributing.rst

```diff
@@ -177,3 +177,27 @@ and `Graphviz <http://graphviz.org>`_ installed. Documentation can be built by r
    make docs
 
 and the resulting HTML will be available at ``docs/build/index.html``.
+
+
+Using live networks
+-------------------
+
+In order to use live networks with `ape`, below environment variables should set:
+
+`WEB3_INFURA_API_KEY`
+`WEB3_ARBITRUM_MAINNET_ALCHEMY_PROJECT_ID`
+`WEB3_ARBITRUM_GOERLI_ALCHEMY_PROJECT_ID`
+`WEB3_OPTIMISM_MAINNET_ALCHEMY_PROJECT_ID`
+`WEB3_OPTIMISM_GOERLI_ALCHEMY_PROJECT_ID`
+
+After setting those, using `ape console --network arbitrum:goerli:alchemy` will work.
+
+
+Publishing and verifying contracts
+----------------------------------
+
+In order to verify contracts `ape-etherscan` plugin used, and below environment variables should set:
+
+`ETHERSCAN_API_KEY`
+`ARBISCAN_API_KEY`
+`OPTIMISTIC_ETHERSCAN_API_KEY`
```

Comment:
I would leave out this part since we don't know the story with verification yet.